internal/time.go

```diff
@@ -33,20 +33,20 @@ func replaceTimes(v interface{}) interface{} {
 
 	m, isMap := v.(map[string]interface{})
 	if isMap {
-		copy := util.CopyMap(m)
+		copyMap := util.CopyMap(m)
```

Comment:
can we call this `mapCopy`? `CopyMap` is more appropriate for method name because it's imperative. In the case of the variable I think `mapCopy` makes it clearer that it is the `copy of the map`